src/auth/strategies/magic-link.strategy.ts

```diff
@@ -36,10 +36,18 @@ export class MagicLinkStrategy extends PassportStrategy(
     let email;
     try {
       email = await this.magicLinkService.getEmailFromHeader(authorization);
-    } catch {
+    } catch (err) {
+      if (err instanceof Error) {
+        return this.fail(`${err.name}: ${err.message}`);
```

Comment:
It feels weird here to fail with either error codes (that are interpreted by the guard) or strings. We should try to be consistent with what is passed to each `fail`